code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -405,6 +405,21 @@ emp_act
 				if (!prob((gotarmor-100)*-1))
 					Paralyse(10)
 
+
+	//Deal damage
+
+	//The on_damaged event returns 1 if the damage should be blocked
+	//Because there are two types of damage at once (brute & burn), do it this way (instead of making a simple if(INVOKE_EVENT) statement)
+	var/damage_blocked = 0
+
+	//INVOKE_EVENT may return null sometimes - this doesn't work nice with bitflags (which is what's being done here). Hence the !! operator - it turns a null into a 0.
+	var/brute_resolved = !!INVOKE_EVENT(on_damaged, list("type" = BRUTE, "amount" = b_loss))
```

Comment:
Holy shit I never thought it'd be able to get away with telling people to use `!!` to make a value a strict bool because nitpicking and yet here you are you absolutely genius.